docs/production.md

```diff
@@ -2,30 +2,26 @@
 
 ## Options
 
 Defaults are chosen in a moderate way between security and performance on a typical machine.
 
-- `maxConnections` is important to prevent the server from going out of memory in combination with maximum body and header size and (for HTTP/2) `maxConcurrentStreams` option.
-- `maxBodySize` is recommended to be set to the lowest necessary for your application. If it is too high, people may fill your memory with useless data. (It is always possible to increase it at runtime, see [usage of Request::getBody($size)](classes/request#getbodyint-ampbytestreammessage).)
-- `maxHeaderSize` should never need to be touched except if you have 50 KB of cookies ...
-- `softStreamCap` is a limit where `Response::write()` returns an unresolved Promise until buffer is empty enough again. If you do not have much memory, consider lowering it, if you have enough, possibly set it a bit higher. It is not recommended to have it higher than available memory divided by `maxConnections` and `maxStreams` and 2 (example: for 8 GB memory, 256 KB buffer is fine). [Should be a multiple of `outputBufferSize`.]
-- `maxConcurrentStreams` is the maximum of concurrent HTTP/2 streams on a single connection. Do not set it too high (but neither too low to not limit concurrency) to avoid trivial attacks.
-- `maxFramesPerSecond` is the maximum number of frames a HTTP/2 client may send per second before being throttled. Do not set it too high (but neither too low to not limit concurrency) to avoid attacks consisting of many tiny frames.
-- `maxInputVars` limits the number of input vars processed by `Response` and `BodyParser`. This is especially important to be small enough in order to prevent HashDos attacks and overly much processing.
-- `maxFieldLen` limits field name lengths in order to avoid excessive buffering (which would defeat any possibilities of incremental parsing).
-- `ioGranularity` is buffering input and output - data will be usually sent after the defined amount of buffered data and fed into the `Body` after receiving at least that amount of data.
-- `maxRequestsPerConnection` limits the number of requests before the connection is closed (for a HTTP/1.1 connection it also controls the Keep-Alive header). Setting it to `PHP_INT_MAX` allows for an unlimited amount of connections.
-- `socketBacklogSize` is the queue size of sockets pending acceptance (i.e. being in queue for an `accept()` call by the `Server`).
-- `deflateContentTypes` is a regular expression containing the content-types of responses to be deflated. If you use a bit more exotic content-types for deflatable content not starting with `text/` or ending with `/xml` or `+xml` or equal to `application/(json|(x-)?javascript)`, you should extend the regex appropriately.
+- `debug` logs debugging-related information and is disabled by default. To enable it, make sure you run HTTP server with debug mode and set log level to `DEBUG`.
```

Comment:
As these are private properties, I'd just write them as normal text, so _"In debug mode …"_, _"The connection limit …"_